### DISCUSSION TITLE
Question about server lag and VIEW_DISTANCE

### DISCUSSION THREAD

**Discussion by mario-hess** [itzg/docker-minecraft-server#2509] [General]
The server runs on a Raspberry Pi 4. When players are far away of each other, it lags (very noticable when driving on rails).
I tried different values for MEMORY but it doesn't seem to do anything.
Is there a known solution to this?
Also I'd like to ask what the default value for VIEW_DISTANCE is, because it is empty in the docs.
Thank you very much!

**Comment by itzg** [comment]
Check to make sure your physical disk I/O isn't maxed out. I believe I/O can make a big difference with various chunks being loaded concurrently. 

As for defaults, they will be the ones here 

https://minecraft.wiki/w/Server.properties

So, view distance defaults to 10.
